Gopkg.lock

```diff
@@ -234,13 +215,95 @@
     "pkg/raft_serverpb",
     "pkg/tikvpb"
   ]
-  revision = "5ef8fa297ea3547b015ef5069cb500bba67b0d94"
+  revision = "279515615485b0f2d12f1421cc412fe2784e0190"
 
 [[projects]]
   branch = "master"
   name = "github.com/pingcap/pd"
   packages = ["pd-client"]
-  revision = "ea6597638eb7f3d7d50248b90e60888352ac8e85"
+  revision = "ba75bdda52f50df752c42c3f1434d9471e98f75e"
+
+[[projects]]
+  branch = "master"
+  name = "github.com/pingcap/tidb"
```

Comment:
very strange that tidb contains tidb repo....

Reply:
+1

Reply:
@arthurkiller 

The tidb should not be in lock file too. 

Maybe you can use `dep ensure` to remove this?

Reply:
```bash
dep ensure -update github.com/pingcap/kvproto
dep ensure -update github.com/pingcap/pd
./hack/clean_vendor.sh
```
I try to update the venor using the above command, the tidb wasn't added in the `Gopkg.lock`.

Reply:
thanks

Reply:
BTW. I did the same...  :(

I will clean the work space and do it again

Reply:
@liukun4515 are you sure do things above will not add `tidb` into vendor ?

Reply:
If updating vendor is a troublesome thing for you, we can give a pr to update vendor, and you can implement getPrevregion base on the latest master.
By the way, if TiDB wants to use the  `getPrevReigon` feature, we should add this feature to [region_cache.go](https://github.com/pingcap/tidb/blob/master/store/tikv/region_cache.go).

Reply:
Maybe it's not time for me to do contribute.XD

Could you pls put up an pr only update things you mentioned?

Reply:
Ahaa, I get it. I run it without run `checkout-pr-branch.sh`

so dep add the github.vendor for me.

I will redo again...